deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -459,7 +459,7 @@ <h4 class="panel-title pull-left pointer-cursor">
                     return {
                         value: item.id,
                         text: item.abstract_name + " (" + item.core + " cores, " + item.mem + " GB, " + item.storage + ", " + item.provider + ": " + item.provider_name + ")",
-                        isSelected: item.abstract_name === currentCluster.hostType
+                        isSelected: item.id === currentCluster.hostType
```

Comment:
MIght want to check if we need to fix this at other places as well.

Reply:
I don't think so, I will test this since we know about this issue, hopefully there aren't too many other aliases for these in the codebase:
```💡 grep -rli "abstract_name.*host.*type" .
./get_host_type.tmpl
./cluster_configuration.html```

Reply:
Hm, I was able to find some in cluster configuration in the template directory, will check those out:
```
💡 pwd
/home/trobbins/code/teletraan/deploy-board/deploy_board/templates/clusters
💡 grep -rli "abstract.*host.*type" ..
../clusters/get_host_type.tmpl
../clusters/cluster_configuration.html
../configs/new_capacity.html
../configs/new_capacity_adv.html
```